Shared/UI/Views/MessageContentView/MessageContentView+Menu.swift

```diff
@@ -46,13 +46,13 @@ extension MessageContentView: UIContextMenuInteractionDelegate {
                                 children: [confirmDelete, neverMind])
 
         let viewReplies = UIAction(title: "View Replies") { [unowned self] action in
-            let item = ConversationMessageItem(channelID: cid, messageID: message.id)
+            let item = MessageSequenceItem(channelID: cid, messageID: message.id)
```

Comment:
The message timeline datasource item has a more generic name now. This is because it now supports conversations and threads.